packages/yuudachi/src/Constants.ts

```diff
@@ -42,7 +42,7 @@ export const REPORT_REASON_MAX_LENGTH = 1_500;
 export const REPORT_REASON_MIN_LENGTH = 10;
 
 export const REPORT_DUPLICATE_PRE_EXPIRE_SECONDS = 3;
-export const REPORT_DUPLICATE_EXPIRE_SECONDS = 15 * 60;
+export const REPORT_DUPLICATE_EXPIRE_SECONDS = 60 * 60;
```

Comment:
Feeling this is still too small, maybe 3 hours?

```suggestion
export const REPORT_DUPLICATE_EXPIRE_SECONDS = 3 * 60 * 60;
```

Reply:
I think even 12 hours would be ok